src/Integration.TeamExplorer.UnitTests/SectionControllerTests.cs

```diff
@@ -51,11 +51,10 @@ public class SectionControllerTests
         [TestInitialize]
         public void TestInitialize()
         {
-            ThreadHelper.SetCurrentThreadAsUIThread();
             this.serviceProvider = new ConfigurableServiceProvider(assertOnUnexpectedServiceRequest: false);
 
             this.sonarQubeServiceMock = new Mock<ISonarQubeService>();
-            this.host = new ConfigurableHost(this.serviceProvider, Dispatcher.CurrentDispatcher)
```

Comment:
Most of the changes are to fix up the tests since the host no longer needs a dispatcher.
The product code changes are very small.